app/controllers/two_factor_authentication/options_controller.rb

```diff
@@ -33,8 +33,8 @@ def process_valid_form
         'personal_key' => login_two_factor_personal_key_url,
         'sms' => otp_send_url(otp_delivery_selection_form: { otp_delivery_preference: 'sms' }),
         'auth_app' => login_two_factor_authenticator_url,
-        'piv_cac' => login_two_factor_piv_cac_url,
-        'webauthn' => login_two_factor_webauthn_url,
+        'piv_cac' => FeatureManagement.piv_cac_enabled? ? login_two_factor_piv_cac_url : nil,
```

Comment:
What if we did 

```ruby
factor_to_url['piv_cac'] = login_two_factor_piv_cac_url if FeatureManagement.piv_cac_enabled?`
```

That helps keep us from having to use a confusing ternary inside a hash

Reply:
reek complains

Reply:
This will be changed up when #2492 gets merged, so I'm not as worried about the ternary.